UnitsNet/UnitDefinitions/Flow.json

```diff
@@ -47,6 +59,66 @@
         }
       ]
     },
+    {
+      "SingularName": "CubicFootPerMinute",
+      "PluralName": "CubicFeetPerMinute",
+      "FromUnitToBaseFunc": "x/2118.88000326",
+      "FromBaseToUnitFunc": "x*2118.88000326",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "ft³/min" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "CubicFootPerHour",
+      "PluralName": "CubicFeetPerHour",
+      "FromUnitToBaseFunc": "x*7.8657907199999087346816086183876e-6",
+      "FromBaseToUnitFunc": "x/7.8657907199999087346816086183876e-6",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "ft³/h", "cf/hr" ]
```

Comment:
Is `cf/hr` used? Having it here means you expect to parse it.
If we keep this, should the same be added for cubic yards per hour below?

Reply:
`cf/hr` was already there before this PR so I didn't remove it. I suggest we put that task of removing it in the wishlist for breaking changes.

Reply:
Ah I missed that. Good point, if you could add it to the wishlist that would be good!

Reply:
Done! :)